### ISSUE TITLE
Geany crashes on Project > Open

### ISSUE THREAD

**Issue by Laurent3170** [geany/geany-plugins#578]
Moving issue from https://github.com/geany/geany/issues/1509 

Using Ubuntu 16.04 and Geany 1.30.1, when I have an opened project and I want to open a new one, I go under Project > Open then it displays the dialog box, and crashes 1-2 seconds after by closing its window.
When I use Recent projects it works well, but the list is not long enough to keep all my projects.
Track traces from gdb
```
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/i386-linux-gnu/libthread_db.so.1".
[New Thread 0xb62ceb40 (LWP 12545)]
[New Thread 0xb58ffb40 (LWP 12546)]
[New Thread 0xb50feb40 (LWP 12547)]
[New Thread 0xaf7ebb40 (LWP 12550)]
[New Thread 0xaeec6b40 (LWP 12551)]
[New Thread 0xab99ab40 (LWP 12554)]
[New Thread 0xab199b40 (LWP 12555)]
[New Thread 0xaa998b40 (LWP 12556)]
[New Thread 0xaa197b40 (LWP 12557)]
[New Thread 0xa9996b40 (LWP 12558)]
[New Thread 0xa9195b40 (LWP 12559)]
[New Thread 0xa8994b40 (LWP 12560)]
openjdk version "1.8.0_131"
OpenJDK Runtime Environment (build 1.8.0_131-8u131-b11-0ubuntu1.16.04.2-b11)
OpenJDK Server VM (build 25.131-b11, mixed mode)

(geany:12544): Pango-WARNING **: Invalid UTF-8 string passed to pango_layout_set_text()
[New Thread 0xa6ce9b40 (LWP 12585)]
[New Thread 0xa644bb40 (LWP 12607)]
[New Thread 0xa5c4ab40 (LWP 12608)]
[Thread 0xa5c4ab40 (LWP 12608) exited]
[Thread 0xa6ce9b40 (LWP 12585) exited]

Thread 1 "geany" received signal SIGSEGV, Segmentation fault.
0xa7098c77 in ?? () from /usr/lib/i386-linux-gnu/geany/projectorganizer.so
```

**Comment by codebrainz** [comment]
Is it #549, #550?

**Comment by Laurent3170** [comment]
@codebrainz Yes it looks like it is, as I have tried to only open a project without one being already opened and it worked, so the problem occurs when closing and not opening.
I just need to wait for a new version available on the ppa.
Thanks

**Comment by saiballo** [comment]
same problem here: 

ubuntu 16.04
geany 1.30.1

**Comment by elextr** [comment]
@saiballo 1.30.1 is too old to contain the fix, needs Geany-plugins 1.31
